src/data/data.go

```diff
@@ -17,23 +20,43 @@ var (
 
 type Repositories struct {
 	Tests TestRepository
+	Users UserRepository
 }
 
-type Context struct {
+type persistenceContext struct {
 	db           *sql.DB
 	config       *config.Config
+	logger       *log.Logger
 	repositories *Repositories
 }
 
-func New(db *sql.DB, config config.Config) Repositories {
+func New(config config.Config, logger *log.Logger) Repositories {
+	logger.Println("Initializing database connections...")
+	db, err := database.Open(config.Db)
+	if err != nil {
+		logger.Fatal(err)
+	}
+	logger.Println("database connection pool established")
+
+	logger.Println("Testing LDAP connection...")
```

Comment:
This is something that I've noticed with me as well. We should look that we design the log messages as uniform as possible. 

Maybe this should get out: 
```suggestion
	logger.Println("Initializing database connections...")
	db, err := database.Open(config.Db)
	if err != nil {
		logger.Fatal(err)
	}

	logger.Println("Testing LDAP connection...")
```

Reply:
What did change there? Is it just the "database connection pool established" line?
I think the topic of log output is too big to discuss here. Let's put it on the agenda for the next meeting.

Reply:
Yes, I have only removed the one log statement. You are right, we should discuss this topic at the next meeting.